.travis.yml

```diff
@@ -1,16 +1,21 @@
 sudo: false
-dist: trusty
```

Comment:
:(

`pypy` installation fails on trusty, at least as of right now. I'd like to have `pypy` and `pypy3` in the allowed failures builds for Travis. If it works on cpython and "another python", it probably works everywhere.